crane_plus_examples/launch/demo.launch.py

```diff
@@ -13,7 +13,9 @@
 # limitations under the License.
 
 from ament_index_python.packages import get_package_share_directory
-from crane_plus_description.robot_description_loader import RobotDescriptionLoader
+from crane_plus_description.robot_description_loader import (
+    RobotDescriptionLoader,
+)
```

Comment:
import部分が文字数制限の緩和の影響を受けて無さそうなので手動で更新が必要そうです。
```suggestion
from crane_plus_description.robot_description_loader import RobotDescriptionLoader
```